app_backend/requirements.txt

```diff
@@ -31,3 +31,4 @@ typing-extensions==3.10.0.2
 urllib3==1.26.7
 uvicorn==0.15.0
 wrapt==1.12.1
+firebase_admin
```

Comment:
Do we need any auth keys to authenticate?

Reply:
Good point. I will add a Firebase project using the Quack Google account.

Reply:
Now I have a private key (a .json file). What's the best way to upload it to the production environment?

Reply:
Just plop it into the project, we can worry about securing our keys later